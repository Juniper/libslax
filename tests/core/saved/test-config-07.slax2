version 1.2;

ns xutil extension = "http://xml.libslax.org/xutil";

main <data> {
    var $slax = xutil:xml-to-slax(., "1.3");
    
    <string> $slax;
    <unescaped> {
        uexpr $slax;
    }
    var $out = xutil:slax-to-xml($slax);
    <out> {
        copy-of $out;
    }
    <more> {
        copy-of xutil:slax-to-xml("<hmmm> 10; <max> 'condition';")/*;
    }
}
