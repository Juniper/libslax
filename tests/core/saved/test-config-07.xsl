<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xutil="http://xml.libslax.org/xutil" version="1.0" extension-element-prefixes="xutil">
  <xsl:template match="/">
    <data>
      <string>
        <xsl:value-of select="xutil:xml-to-slax(., &quot;1.3&quot;)"/>
      </string>
      <unescaped>
        <xsl:value-of select="xutil:xml-to-slax(., &quot;1.3&quot;)" disable-output-escaping="yes"/>
      </unescaped>
    </data>
  </xsl:template>
</xsl:stylesheet>
