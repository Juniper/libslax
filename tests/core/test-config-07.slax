version 1.3;

main <data> {
    <string> xutil:xml-to-slax(., "1.3");
    <unescaped> {
        uexpr xutil:xml-to-slax(., "1.3");
    }
}
