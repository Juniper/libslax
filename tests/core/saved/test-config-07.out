<?xml version="1.0"?>
<data>
  <string>&lt;configuration&gt; {
    &lt;system&gt; {
        &lt;host-name&gt; "puff-the-magic-dragon";
        &lt;domain-name&gt; "by-the-sea.example.com";
        &lt;name-server&gt; {
            &lt;name&gt; "10.1.2.3";
        }
    }
    &lt;protocols&gt; {
        &lt;bgp&gt; {
            &lt;metric&gt; "15";
        }
    }
    &lt;interfaces&gt; {
        &lt;interface&gt; {
            &lt;name&gt; "xe-1/1/1";
        }
        &lt;interface&gt; {
            &lt;name&gt; "ge-1/1/1";
        }
    }
}
</string>
  <unescaped><configuration> {
    <system> {
        <host-name> "puff-the-magic-dragon";
        <domain-name> "by-the-sea.example.com";
        <name-server> {
            <name> "10.1.2.3";
        }
    }
    <protocols> {
        <bgp> {
            <metric> "15";
        }
    }
    <interfaces> {
        <interface> {
            <name> "xe-1/1/1";
        }
        <interface> {
            <name> "ge-1/1/1";
        }
    }
}
</unescaped>
</data>
